src/main.rs

```diff
@@ -16,6 +16,7 @@ use crate::util::{choose, read_yesno, write_tsv};
 mod download;
 mod logger;
 mod rename;
+use rename::RenameAction;
```

Comment:
Could you please:

1. Put this `use` up with the `use::crate` statements above? In fact, please just merge it with `use crate::rename::RenameBuilder;`.
2. Please import `RenameAction` unqualified to avoid the stutter when writing out `rename::RenameAction`.

Reply:
:+1: